scripts/deploy_from_template.sh

```
@@ -17,7 +17,11 @@ else
     TAG="latest"
 fi
 
-oc process -p IMAGE=$IMAGE -p IMAGE_TAG=$TAG -p INVENTORY_URL="https://api.stage.openshift.com/api/assisted-install/v2" -f template.yaml --local | oc apply -n $NAMESPACE -f -
+oc process -p IMAGE=$IMAGE \
+           -p IMAGE_TAG=$TAG \
+           -p INVENTORY_URL="https://api.stage.openshift.com/api/assisted-install/v2" \
+           -p PULL_SECRET_URL="https://api.stage.openshift.com/api/accounts_mgmt/v1/access_token"
+           -f template.yaml --local | oc apply -n $NAMESPACE -f -
```

Shellcheck (0.10.0)

[warning] This flag is used as a command name. Bad line break or missing [ .. ]?

(SC2215)